ComfortZoneApp/app/src/main/java/com/example/comfortzone/utils/ComfortCalcUtil.java

```diff
@@ -59,19 +62,27 @@ private static int getSum(LevelsTracker tracker) {
 
     public static void calculateAverages(ParseUser currentUser) {
         ArrayList<LevelsTracker> trackerArrayList = (ArrayList<LevelsTracker>) currentUser.get(KEY_LEVEL_TRACKERS);
-        trackerArrayList.forEach(tracker -> calculateSingularAverage(tracker));
+        List<Task<Void>> savingAverages = trackerArrayList.stream().filter(tracker -> tracker.getCount() > 0).map(tracker -> calculateSingularAverage(tracker)).collect(Collectors.toList());
```

Comment:
@fionamei can you format this line of code so thats its more readable? 

List<Task<Void>> savingAverages = trackerArrayList.stream().filter(tracker -> tracker.getCount() > 0)
                                                               .map(tracker ->calculateSingularAverage(tracker))
                                                               .collect(Collectors.toList());

Something like this. Its a good practice to not have all code in one line so that user does not have to scroll the editor.